ext/curl/tests/Caddyfile

```diff
@@ -0,0 +1,13 @@
+{
+    admin off
+    auto_https disable_redirects
+}
+
+localhost
```

Comment:
Do you think we could switch to a port that is less likely to clash? E.g. `8888`?